sessions/smoother-collaboration.qmd

```diff
@@ -5,70 +5,76 @@
+    R.  
 
-More specific objectives are to:
+**Specific objectives** are to:
 
 1.  Explain what a "project-oriented" workflow is, what a project-level
-    R dependency management is, and why these concepts are important to
-    consider in collaborative and reproducible analyses.
-2.  Describe the difference between "workflow dependencies" and "build
-    dependencies".
-3.  Apply functions in the `{usethis}` R package to implement these
-    dependency management concepts.
-4.  Explain the role that following a style guide has on building a
-    common approach to reading (and writing) R code, and thus improve
-    project-level collaboration.
-5.  Use `{styler}` and RStudio's canonical markdown mode to
+    R dependency management is, and why these concepts are important for
+    collaborative and reproducible analyses.
+2.  Describe the difference between workflow dependencies and build
+    dependencies, and apply functions in the `{usethis}` R package to
+    implement these dependency management concepts.
+3.  Explain how following a style guide helps build a common approach to
+    reading and writing R code, and thereby improves project-level
+    collaboration.
+4.  Use `{styler}` and RStudio's canonical markdown mode to
     programmatically check and apply style guides to your project files.
 
-## :speech_balloon: Discussion activity: How do you *exactly* collaborate or contribute? To your own or others' projects.
+## :speech_balloon: Discussion activity: How do you *exactly* collaborate or contribute to your own or others' projects?
 
 > Time: \~10 minutes.
 
-When you work on a project (for your thesis or a manuscript), how
-**exactly** do you and your collaborators contribute to the project? Is
-it mostly verbal contributions? Do you use a shared folder that the
-files are on? How do you keep track of who's changed what? Do you mostly
-work on your own and contributions are largely verbal or written
-feedback (like in a meeting or through an email)? If you work directly
-on a project, how do you coordinate things? Does one collaborator work
-on one section or analysis, so your files are separate? Do you ever have
-to go in and contribute your own code to theirs (and vice versa)?
-
-1.  Take about 1 minute to think on these questions.
+Reflect on, when you work on a project (for your thesis or a
+manuscript), how **exactly** do you and your collaborators contribute to
+the project:
+
+-   Is it mostly verbal contributions?
+-   Do you use a shared folder that the files are on?
+-   How do you keep track of who's changed what?
+-   Do you mostly work on your own with contributions being mostly
+    verbal or written feedback (like in a meeting or through an email)?
+-   If you collaborate directly on a project, how do you coordinate
+    things? Does one collaborator work on one section or analysis, so
+    your files are separate?
+-   Do you ever have to go in and contribute your own code to theirs
+    (and vice versa)?
+
```

Comment:
```suggestion
    (and vice versa)?

Consider these questions as we do the following steps.
```

Having an unnumbered list beside a numbered list will look a bit odd when rendered, so just putting some text between to separate them.